src/components/Spawn/spawnHostModal/getFormSchema.tsx

```diff
@@ -425,21 +424,21 @@ export const getFormSchema = ({
         "ui:data-cy": "distro-input",
       },
       region: {
-        "ui:widget": AntdSelect,
         "ui:data-cy": "region-select",
         "ui:disabled": isMigration,
         "ui:elementWrapperCSS": dropdownWrapperClassName,
         "ui:valuePlaceholder": "Select a region",
+        "ui:allowDeselect": false,
       },
       publicKeySection: {
         useExisting: {
           "ui:widget": widgets.RadioBoxWidget,
         },
```

Comment:
Could we add a state for no public keys as well?

<img width="257" alt="Screen Shot 2022-12-14 at 4 41 52 PM" src="https://user-images.githubusercontent.com/47064971/207721044-a741aa18-77dc-488b-99ce-aa2ece4c2919.png">

Reply:
<img width="359" alt="image" src="https://user-images.githubusercontent.com/4605522/207726084-70915d08-b60f-4a62-ac51-05bab8123dee.png">